exp/query/api_test.go

```diff
@@ -37,8 +40,109 @@ const (
 
 	// This is pre-generated vectorized query via text2vec-contextionary
 	textVectorized = "./testdata/vectorized.json"
+
+	// Collection name for data in `testdata`
+	testCollection = "Question"
+
+	// testTenant for data in `testdata`
+	// TODO(kavi): Change to something generic
+	testTenant = "kavi"
 )
 
+func TestAPI_propertyFilters(t *testing.T) {
```

Comment:
It's really hard to test `Search` itself, because of it's dependency with offloads3 module (that downloads from s3/minio). Hard to mock.

I separated each search to separate method, so that I can mock the data and test it individually.